### ISSUE TITLE
XDebug instructions / PHP path "seems" to be incorrect

### ISSUE THREAD

**Issue by cameronterry** [10up/wp-local-docker-v2#283]
**Describe the bug**

TLDR:

* The folder path, `/etc/php/7.4/fpm/conf.d/`, provided in the instructions and in the default setup in `docker-compose.yml` appears to be incorrect. The folder seemingly exists at `/etc/php.d/` instead.
* The supplied `docker-php-ext-xdebug.ini` ([link](https://github.com/10up/wp-local-docker-v2/blob/1c10aa43f9787ca0517a90e8a24769fd6c45eeae/src/config/php-fpm/docker-php-ext-xdebug.ini)) seems to be missing several key configuration parameters to work with XDebug 2.x.x in some containers (PHP 7.4 in this ticket and circumstance).

I guess the question comes down to what is correct and incorrect: is the instructions correct and the container's folder structure / PHP install incorrect? Or vice-versa?

Long Version:

I setup a brand new environment (not updating a pre-existing one) using PHP 7.4 and then followed the instructions in [the README to setup XDebug](https://github.com/10up/wp-local-docker-v2/blob/1c10aa43f9787ca0517a90e8a24769fd6c45eeae/README.md?plain=1#L310). 
However XDebug did not work.

After a period of trial and error, I noticed that folder suggested in the README - `/etc/php/7.4/fpm/conf.d/` - does not exist. Using `10updocker shell` and then using `ls -d -l /etc/php*` to look at the folder structure, reveals it to be:

```
$ ls -d -l /etc/php*
drwxr-xr-x 2 www-data root  4096 Jan 24  2021 /etc/php-extensions-available
-rw-rw-rw- 1 www-data root  4920 Jan 17  2021 /etc/php-fpm.conf
drwxr-xr-x 1 www-data root  4096 Jan 17  2021 /etc/php-fpm.d
drwxr-xr-x 1 www-data root  4096 Jan 24  2021 /etc/php-zts.d
drwxr-xr-x 1 www-data root  4096 Feb  7 09:39 /etc/php.d
-rw-r--r-- 1 www-data root 63250 Jan  5  2021 /etc/php.ini
```

Also running the following command (forgive the naivety / simplicity - "Friday brain") show that PHP was configured with another ini file for XDebug.

```
$ php -i | grep "/etc" | grep "xdebug" 
/etc/php.d/15-xdebug.ini,
```

Based on this information, it would seem that the volume / XDebug ini file noted in the README and and what comes with 10updocker as the default (for new environments) `docker-compose.yml` is incorrect. And the container is using `/etc/php.d/` for `.ini` files instead. For me, I had to do the following:

Change this (using `/etc/php/7.4/conf.d/`):
```
'./config/php-fpm/docker-php-ext-xdebug.ini:/etc/php/7.4/fpm/conf.d/99-ext-xdebug.ini:cached'
```
To this (using `/etc/php.d/` path):
```
'./config/php-fpm/docker-php-ext-xdebug.ini:/etc/php.d/99-docker-php-ext-xdebug.ini:cached'
```

After this, I could get XDebug enabled but it still did not work. Checking with the `10updocker shell` again, it seems the XDebug version was 2.9.8:

```
$ php -v | grep -i "xdebug"
    with Xdebug v2.9.8, Copyright (c) 2002-2020, by Derick Rethans
```

Adding the following lines to the `docker-php-ext-xdebug.ini` file (under the `config/` folder) got XDebug up and running.

```
xdebug.remote_autostart = 1
xdebug.remote_enable = on
xdebug.remote_host = host.docker.internal
```

**Environment information**
 - Device: Custom built PC
 - OS: Ubuntu 20.04.3 LTS
 - Docker Desktop version: 20.10.12, build e91ed57

**Additional context**
